x-pack/plugins/apm/public/components/app/service_overview/service_overview_dependencies_table/index.tsx

```diff
@@ -0,0 +1,247 @@
+            valueLabel={asTransactionRate(throughput.value)}
+          />
+        );
+      },
+      sortable: true,
+    },
+    {
+      field: 'error_rate_value',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.dependenciesTableColumnErrorRate',
+        {
+          defaultMessage: 'Error rate',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { error_rate: errorRate }) => {
+        return (
+          <SparkPlotWithValueLabel
+            compact
+            color="euiColorVis7"
+            series={errorRate.timeseries ?? undefined}
+            valueLabel={asPercent(errorRate.value, 1)}
+          />
+        );
+      },
+      sortable: true,
+    },
+    {
+      field: 'impact_value',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.dependenciesTableColumnImpact',
+        {
+          defaultMessage: 'Impact',
+        }
+      ),
+      width: px(unit * 4),
+      render: (_, { impact }) => {
+        return <ImpactBar size="m" value={impact} />;
+      },
+      sortable: true,
+    },
+  ];
+
+  const {
+    urlParams: { start, end, environment },
+  } = useUrlParams();
+
+  const { data = [], status } = useFetcher(() => {
+    if (!start || !end) {
+      return;
+    }
+
+    return callApmApi({
+      endpoint: 'GET /api/apm/services/{serviceName}/dependencies',
+      params: {
+        path: {
+          serviceName,
+        },
+        query: {
+          start,
+          end,
+          environment: environment || ENVIRONMENT_ALL.value,
+          numBuckets: 20,
+        },
+      },
+    });
+  }, [start, end, serviceName, environment]);
+
+  // need top-level sortable fields for the managed table
+  const items = data.map((item) => ({
+    ...item,
+    error_rate_value: item.error_rate.value,
+    latency_value: item.latency.value,
+    throughput_value: item.throughput.value,
+    impact_value: item.impact,
+  }));
+
+  return (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiTitle size="xs">
+              <h2>
+                {i18n.translate(
+                  'xpack.apm.serviceOverview.dependenciesTableTitle',
+                  {
+                    defaultMessage: 'Dependencies',
+                  }
+                )}
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+          <TableLinkFlexItem>
+            <ServiceMapLink serviceName={serviceName}>
+              {i18n.translate(
+                'xpack.apm.serviceOverview.dependenciesTableLinkText',
+                {
+                  defaultMessage: 'View service map',
+                }
+              )}
+            </ServiceMapLink>
+          </TableLinkFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <TableFetchWrapper status={status}>
+          <ServiceOverviewTableContainer
+            isEmptyAndLoading={
```

Comment:
There's a bunch of styling in `ServiceOverviewTableContainer` - don't we want that here as well?

Reply:
Good point. But I'd prefer to keep the custom styling to a minimum. So if we can avoid `isEmptyAndLoading` and the resulting `visibility` styling that would be better.

Also: have we talked to EUI about better ways at getting the styles we want, instead of the custom route?

Reply:
@smith, any thoughts here?

Reply:
`ServiceOverviewTableContainer` makes a fixed-size table and the styling pins the pagination to the bottom. elastic/eui#4281 is open to make it so we don't have to override styles if EUI were to implement this natively.

elastic/eui#4282 was opened by me and closed by the team to handle the empty loading state. The default behavior makes it so there's a cell with text in it while it's loading, which looks much worse than what we have now, where it's empty and shows the loading bar, which looks pretty nice.

Reply:
I'd recommend using the tables as we have them now on the overview and working on making the rest of the tables in APM consistent. I believe @formgeist is doing some work on this.

Reply:
sgtm 👍

Reply:
...but, the `ServiceOverviewTable` was meant to be used for each of the tables so I don't understand why we need to export the container.

Reply:
I'm using a EuiInMemoryTable rather than a EuiBasicTable. What I can do is export _only_ the container, would that be better?

Reply:
> What I can do is export only the container, would that be better?

Yes since we're just passing the props along to the basic or in-memory table that should work.

Reply:
I'll make sure to do that in the service instances PR.